### ISSUE TITLE
Not detecting comments correctly 

### ISSUE THREAD

**Issue by ChrisBoehmCA** [dkelosky/vscode-ibm-jcl#3]
Not sure if this is caused by the , continuation character or not, but in the below example lines 2-5 are not displayed as comments
```
//PROCNAME PROC LEPARM=,                 < OPTIONAL LE PARM
//*  LOGLVL='+I',                        < Set log level
//*  LOGLVL=' '                          < Set log level to none
//*  LOGLVL='+I',                        < Set log level to Info
//*  LOGLVL='+T',                        < Set log level to Trace
//   LOGLVL='+D',                        < Set log level to debug
//*

**Comment by dkelosky** [comment]
Please let me know if fixed on 0.4.0 as well

**Comment by ChrisBoehmCA** [comment]
Thanks this is fixed now, comments are being detected much better
